### PR TITLE
Nexial webez migration v2 to v3

### PR DESCRIPTION
**Pull request by umeshaccion**
Nexial Migration Done and Code review comments addressed.

### CODE REVIEW

File: inspection/background.js [hunk 1]
```diff
@@ -140,12 +223,13 @@ chrome.windows.getAll({
   let i = 0, w = windows.length, currentWindow;
   for( ; i < w; i++ ) {
       currentWindow = windows[i];
-      let j = 0, t = currentWindow.tabs.length, currentWindowTab;
+      currenttabs = getCurrentTab();
+      let j = 0, t = currenttabs.length, currentWindowTab;
       for( ; j < t; j++ ) {
-        currentWindowTab = currentWindow.tabs[j];
+        currentWindowTab = currenttabs[j];
```
Comment (mikeliucc) [inline inspection/background.js]: please rename to `currentTabs` instead of `currenttabs`

File: inspection/background.js [hunk 2]
```diff
@@ -154,27 +238,71 @@ chrome.windows.getAll({
...
       break;
   }
   updateBadge();
+  sendResponse();
+  return true;
 });
+
+chrome.runtime.onMessage.addListener((request, sender, sendResponse)=>{
+  
+  if (request?.callMethod == "start"){
```
Comment (mikeliucc) [inline inspection/background.js]: perhaps using a `switch` statement instead?

File: inspection/contentScript.js [hunk 3]
```diff
@@ -510,7 +511,7 @@ function sendInspectInfo(command, event) {
       selectedLocator: selectedLocator,
     },
   };
-
+  console.log('send inspect info...');
```
Comment (mikeliucc) [inline inspection/contentScript.js]: please change/remove the use of `console.log` to using `sendConsole`

File: resources/scripts/console.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ function sendConsole(level, logging, data = null) {
   };
   if (APP_ENV === "development") {
     printLog(level, logging, data);
+    console.log(chrome);
```
Comment (mikeliucc) [inline resources/scripts/console.js]: not needed?

File: manifest.json [hunk 5]
```diff
@@ -17,27 +12,31 @@
...
   "icons":                    {
     "16":  "resources/img/nexial-16.png",
     "32":  "resources/img/nexial-32.png",
     "64":  "resources/img/nexial-64.png",
     "128": "resources/img/nexial-128.png"
   },
-  "manifest_version":         2,
+  "manifest_version":         3,
   "name":                     "Nexial WebEZ",
   "version":                  "0.9.1",
```
Comment (mikeliucc) [inline manifest.json]: Please update to 1.0.0

File: view/popup.js [hunk 6]
```diff
@@ -144,27 +121,37 @@ startInspect.addEventListener("click", function () {
...
+  chrome?.storage?.local.get(['inspectList'],(result)=>{
+
+    if(result?.inspectList?.length > 0)
+    {
+      const response = confirm("There are some steps already captured once you start inspecting it will clear eariler steps.?");
+      if(response)
+      {
+        start();
+        $('#inspect_table').remove();
+        chrome.runtime.sendMessage({callMethod: 'start'}, (response)=> {});
```
Comment (mikeliucc) [inline view/popup.js]: please create constants for common literals, like `'start'`, `'stop'`, `'click'`, etc.

**Review by mikeliucc — COMMENTED** [review]
Please review comments and make necessary adjustments

File: inspection/background.js (hunk 1 above)
Comment (umeshaccion) [reply inspection/background.js]: Done with this change and pushed

File: inspection/background.js (hunk 2 above)
Comment (umeshaccion) [reply inspection/background.js]: Done with this change and pushed

File: inspection/contentScript.js (hunk 3 above)
Comment (umeshaccion) [reply inspection/contentScript.js]: Done with this change and pushed

File: resources/scripts/console.js (hunk 4 above)
Comment (umeshaccion) [reply resources/scripts/console.js]: Done with this change and pushed

File: manifest.json (hunk 5 above)
Comment (umeshaccion) [reply manifest.json]: Done with this change and pushed

File: view/popup.js (hunk 6 above)
Comment (umeshaccion) [reply view/popup.js]: Done with this change and pushed

File: manifest.json [hunk 7]
```diff
@@ -17,27 +12,31 @@
...
+  "manifest_version":         3,
   "name":                     "Nexial WebEZ",
-  "version":                  "0.9.1",
-  "permissions":              ["activeTab", "contextMenus"],
-  "web_accessible_resources": ["resources/style/nexial.css", "view/dialogInjector.js"]
+  "version":                  "1.0.0",
+  "permissions": ["activeTab","contextMenus","storage"],
+  "web_accessible_resources":[
+      {
+        "resources": ["resources/style/nexial.css", "view/dialogInjector.js","env.js","resources/scripts/console.js","inspection/contextMenu.js","*.css"],
```
Comment (mikeliucc) [inline manifest.json]: Do we still need `dialogInjector.js`?

File: manifest.json [hunk 8]
```diff
@@ -17,27 +12,31 @@
...
     "128": "resources/img/nexial-128.png"
   },
-  "manifest_version":         2,
+  "manifest_version":         3,
   "name":                     "Nexial WebEZ",
-  "version":                  "0.9.1",
-  "permissions":              ["activeTab", "contextMenus"],
-  "web_accessible_resources": ["resources/style/nexial.css", "view/dialogInjector.js"]
+  "version":                  "1.0.0",
+  "permissions": ["activeTab","contextMenus","storage"],
```
Comment (mikeliucc) [inline manifest.json:35]: is "storage" permission needed?

File: manifest.json (hunk 7 above)
Comment (umeshaccion) [reply manifest.json]: Removed dialogInjector.js file and pushed the changes

File: manifest.json (hunk 8 above)
Comment (umeshaccion) [reply manifest.json:35]: Storage permission is required to access chrome.storage

**Pull request merged**